source/include/cellular_api.h

```diff
@@ -639,6 +639,78 @@ CellularError_t Cellular_SocketRegisterClosedCallback( CellularHandle_t cellular
                                                        CellularSocketClosedCallback_t closedCallback,
                                                        void * pCallbackContext );
 
+/**
+ * @brief Configure parameters of an SSL Context.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] sslContextId The SSL context ID.
+ * @param[in] sslConfigurationParameter The SSL parameter to be configured.
+ * @param[in] inputArg The value to be passed to the SSL parameter.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_ConfigureSSLContext(CellularHandle_t cellularHandle,
+                                      uint8_t sslContextId,
+                                      const char* sslConfigurationParameter,
+                                      const char* inputArg);
+
+/**
+ * @brief Delete a File in the Storage.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] filename Name of the file to be deleted.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_DeleteFileFromStorage(CellularHandle_t cellularHandle, 
+                                            const char* filename);
+
+/**
+ * @brief  Lists the information of a single file.
+ *
+ * @param[in] cellularHandle The opaque cellular context pointer created by Cellular_Init.
+ * @param[in] filename Name of the file to be listed.
+ *
+ * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
+ * code indicating the cause of the error.
+ */
+CellularError_t Cellular_VerifyFileFromStorage(CellularHandle_t cellularHandle, 
```

Comment:
I can't understand what this function does after reading the docs and looking at its declaration. is it checking if the file exists? If so, change its name to "Cellular_CheckFileExists" perhaps. And update the documentation to be a bit more obvious